src/main/java/frc/robot/OI.java

```diff
@@ -108,7 +108,9 @@ public OI() {
 
       d_btn_LB.whenPressed(new LaunchHatch());
       d_btn_A.whenActive(new BeakSet());
+      d_btn_RB.whenPressed(new ToggleSmartIntake());
```

Comment:
Just an FYI that we'll need move this to the A-B-X-Y button to prevent mispresses for the trigger.

Also, expect to move all CargoIntake commands to the operaterOI and not the driverOI in the future.